### PR TITLE
Remove Travis token, which is not needed for open source repos

### PR DESCRIPTION
**Pull request by onetechnical**
## Summary

A Travis token was included for a build status image, but this token is not needed for open source repositories.

## Test Plan

Just ensure build image continues to work for README.md and is removed from buildhost/README.md.

### CODE REVIEW

File: scripts/buildhost/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/algorand/go-algorand.svg?token=25XP72ADqbCQJ3TJVC9S&branch=master)](https://travis-ci.com/algorand/go-algorand)
+[![Build Status](https://travis-ci.com/algorand/go-algorand.svg?branch=master)](https://travis-ci.com/algorand/go-algorand)
```
Comment (tsachiherman) [inline scripts/buildhost/README.md:1]: this line is not needed. The build status is irrelevant on this file.
( we don't compile the buildhost directory, so there is no "build status" here )

**Review by tsachiherman — CHANGES_REQUESTED** [review]
it makes sense to fix this on the root `READ.me`, but not on the `./scripts/buildhost/README.md`

**Review by tsachiherman — APPROVED** [review]

File: scripts/buildhost/README.md (hunk 1 above)
Comment (onetechnical) [reply scripts/buildhost/README.md]: That is a very good point. Fixed that and a formatting bug.

**Comment by tsachiherman** [comment]
Last failure was caused due to a bug in one of the unit tests. I just fixed it, so you might want to rebase your branch if this build keep failing.

**Pull request merged**